src/main/java/de/hysky/skyblocker/skyblock/museum/PriceData.java

```diff
@@ -0,0 +1,48 @@
+package de.hysky.skyblocker.skyblock.museum;
+
+import de.hysky.skyblocker.utils.ItemUtils;
+import net.minecraft.util.Pair;
+
+public class PriceData {
+	private double lBinPrice = 0;
+	private double craftCost = 0;
```

Comment:
The craft cost and bin price fields should be marked as `final` if they can be since I don't see them assigned outside of the constructor